packages/pds/src/app-view/api/app/bsky/feed/getActorLikes.ts

```diff
@@ -0,0 +1,135 @@
+import { InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../../../../lexicon'
+import { FeedKeyset } from '../util/feed'
+import { paginate } from '../../../../../db/pagination'
+import AppContext from '../../../../../context'
+import { FeedRow } from '../../../../services/feed'
+import { OutputSchema } from '../../../../../lexicon/types/app/bsky/feed/getAuthorFeed'
+import { handleReadAfterWrite } from '../util/read-after-write'
+import { authPassthru } from '../../../../../api/com/atproto/admin/util'
+import { LocalRecords } from '../../../../../services/local'
+
+export default function (server: Server, ctx: AppContext) {
+  server.app.bsky.feed.getActorLikes({
```

Comment:
good news: we're incredibly close to not having to re-implement these on the PDS 👀